src/lib/mapper.service.ts

```diff
@@ -145,7 +145,7 @@ export class NGXMapperService {
       NGXMapperService.getPosition(stackLine),
       NGXMapperService.getMapFilePath(stackLine)
     ]).pipe(
-      switchMap<[LogPosition, string], LogPosition>(([distPosition, sourceMapLocation]) => {
+      switchMap<[LogPosition, string], Observable<LogPosition>>(([distPosition, sourceMapLocation]) => {
```

Comment:
This needs to be reverted. Looking at RxJS definition for switchmap `export function switchMap<T, R>(project: (value: T, index: number) => ObservableInput<R>): OperatorFunction<T, R>;`. The value return is `ObservableInput<R>`. So with the typing you added here we would be returning `Observable<Observable<LogPosition>>`

Reply:
if i use yarn install, it installs rxjs v6.5.2. There the second parameter has to be `ObservableInput<any>`.

see:
https://github.com/ReactiveX/rxjs/blob/b8d568bccc8c8687fd163932b9f2ce52cf499614/src/internal/operators/switchMap.ts#L13-L17

Reply:
ahh. Interesting. I guess I'll need to tighten the constraint on rxjs since there was a breaking typing introduced in a minor version bump.

Reply:
thats why i've added the yarn lock, so the build deps are pinned